compiler/il/OMRBlock.cpp

```diff
@@ -1708,6 +1708,19 @@ OMR::Block::getGlobalNormalizedFrequency(TR::CFG * cfg)
    return frequency;
    }
 
+/*
+ * An OSR induce block is used in postExecutionOSR to contain the OSR point's dead stores
+ * and to induce the transition. It will always have no successors and one exception
+ * successor to an OSR catch block.
+ */
+bool
+OMR::Block::isOSRInduceBlock(TR::Compilation *comp)
+   {
+   return comp->getOSRMode() == TR::voluntaryOSR
+      && self()->getExceptionSuccessors().size() == 1
```

Comment:
right, I think Vijay's point is that the block could potentially have other exception successors (ie real ones) and we should account for that. Perhaps this could read more like:
`if (comp->getOSRMode() != TR::voluntaryOSR)
   return false;
if (self()->getExcetionSuccessors().size() == 1
   && ,,,(
   return true;
// loop to check for an OSRCatch exception successor`